### PR TITLE
UserGroup from @bricks/harmony

### LINKED ISSUE #2571: UserGroup from `@bricks/harmony`

### PR DESCRIPTION
**Pull request by asabotovich**
## PR includes

- [x] Refactor

## Related issues

Resolve #2571

### CODE REVIEW

**Comment by asabotovich** [comment]
Снимок экрана 2024-05-20 в 17 18 23

**Comment by asabotovich** [comment]
Снимок экрана 2024-05-20 в 17 19 32

File: package.json [hunk 1]
```diff
@@ -49,7 +49,7 @@
     "@sentry/nextjs": "7.99.0",
     "@tanstack/react-query": "4.29.5",
     "@tanstack/react-query-devtools": "4.29.6",
-    "@taskany/bricks": "5.31.2",
+    "@taskany/bricks": "0.0.0-5b0130e",
```
Comment (DenisVorop) [inline package.json:52]: be careful

File: src/components/GoalListItemCompact/GoalListItemCompact.tsx [hunk 2]
```diff
@@ -142,7 +142,7 @@ const Column: ColumnRender = ({ col, componentProps }) => {
             content = nullable(projectId, (id) => <TableRowItemText>{id}</TableRowItemText>);
             break;
         case 'issuers':
-            content = nullable(issuers, (list) => <UserGroup users={list} />);
+            content = nullable(issuers, (list) => <UserGroup users={list.map(safeUserData).filter(Boolean)} />);
```
Comment (DenisVorop) [inline src/components/GoalListItemCompact/GoalListItemCompact.tsx:145]: `useMemo`?

Comment (DenisVorop) [reply src/components/GoalListItemCompact/GoalListItemCompact.tsx:145]: And in other places as well

**Review by nyet-ty — APPROVED** [review]

**Pull request merged**